### PR TITLE
#165345758 Implement functionality to get user's stats

### PR DESCRIPTION
**Pull request by kleva-j**
#### What does this PR do?
- Implements the functionality to get stats which includes the number of users article, the number of the users' followers and the number of authors the user follows. 

#### Description of Task to be completed?
- Create endpoint to get the number of a user's followers and followings.
- Create endpoint to get the number of articles created by a user.

#### How should this be manually tested?
- Make a `GET` request to this endpoint `/api/v1/user/followers$followings` to get a user's followers and followings.
- Make a `GET` request to this endpoint `/api/v1/user/articlescount` to get the number of articles created by a user.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[165345758](https://www.pivotaltracker.com/story/show/165345758)

#### Questions:

### CODE REVIEW

**Review by youngestdj — APPROVED** [review]

**Review by olajide1234 — APPROVED** [review]

File: controllers/follow.js [hunk 1]
```diff
@@ -70,4 +71,58 @@ export default class FollowController {
...
+
+  /**
+   * @description Get user's followers and followings stats
+   * @param {object} req http request object
+   * @param {object} res http response object
+   * @returns {object} response
+   */
+  static async getUserFollowers(req, res) {
+    const { id } = req.user;
+    const requiredOutput = {
```
Comment (9jaswag) [inline controllers/follow.js]: @kleva-j since these are attributes, is `requiredOutputs` a better variable name?

File: controllers/user.js [hunk 2]
```diff
@@ -260,4 +260,26 @@ export default class UserController {
...
+   * @param {object} res http response object
+   * @returns {object} response
+   */
+  static async getUserArticlesCount(req, res) {
+    const { id: userId } = req.user;
+    try {
+      const { count } = await Article.findAndCountAll({ where: { userId } });
+      return res.status(200).json({
+        success: true,
+        numberOfArticlesCreated: count
```
Comment (9jaswag) [inline controllers/user.js]: @kleva-j can you come up with a better name here?

**Review by 9jaswag — CHANGES_REQUESTED** [review]

File: controllers/follow.js (hunk 1 above)
Comment (kleva-j) [reply controllers/follow.js]: Thanks, I'll make changes to this.

File: controllers/user.js (hunk 2 above)
Comment (kleva-j) [reply controllers/user.js]: Ok, thanks.

**Review by 9jaswag — APPROVED** [review]

**Pull request merged**